server/app/controllers/DisplayableMessage.java

```diff
@@ -0,0 +1,39 @@
+package controllers;
+
+import views.components.ToastMessage.ToastType;
+
```

Comment:
Please add a javadoc string for the type as well as the nested enum.

Reply:
Thanks will do